src/04-reverseString/main.js

```diff
@@ -1,12 +1,27 @@
 function reverseStringV1(str) {
   // Using for loop:
+  let reverseStr = "";
+  for (let index = str.length; index >= 0; index--) {
+    reverseStr = reverseStr + str.charAt(index);
+  }
+  return reverseStr;
 }
 
 function reverseStringV2(str) {
   // Using arrays:
+  return str.split("").reverse().join("");
+}
+
+function reverseStringV3(str) {
```

Comment:
We will talk about recursive approach in the upcoming session

Reply:
Grateful
I look forward to the next session

Reply:
Thanks